ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb

```diff
@@ -13,6 +13,15 @@
     "Let's start by importing required libraries and initializing session and other variables used in this notebook. By default, the notebook uses the default Amazon S3 bucket in the same AWS Region you use to run this notebook. If you want to use a different S3 bucket, make sure it is in the same AWS Region you use to complete this tutorial, and specify the bucket name for `bucket`. "
```

Comment:
<p>"<code>AmazonSageMakerFullAccess</code>: If you do not require granular permissions for your use case, you can attach the<code><a href="https://console.aws.amazon.com/iam/home?#/policies/arn:aws:iam::aws:policy/AmazonSageMakerFullAccess" target="_blank">AmazonSageMakerFullAccess</a></code> policy to your IAM user or role. If you are running this example in a SageMaker notebook instance, this is the IAM execution role used to create your notebook instance. If you need granular permissions, see <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-security-permission.html#sms-security-permissions-get-started" target="_blank">Assign IAM Permissions to Use Ground Truth</a> for a granular policy to use Ground Truth."</p><p> </p><p>"The AWS managed policy <code><a href="https://console.aws.amazon.com/iam/home#policies/arn:aws:iam::aws:policy/AmazonSageMakerGroundTruthExecution" target="_blank">AmazonSageMakerGroundTruthExecution</a></code>: Run the following code snippet to see your IAM execution role name. This <a href="https://github.com/aws/amazon-sagemaker-examples/raw/a0b65199199cb57249093e85a9042e4898afbbcc/ground_truth_labeling_jobs/video_annotations_quality_assessment/images/add-policy-loop.gif" target="_blank">GIF</a> demonstrates how to attach this policy to an IAM role in the IAM console. For further instructions, see the <a href="https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html#add-policies-console" target="_blank">Adding and removing IAM identity permissions</a> section in the <em>AWS Identity and Access Management User Guide</em>."</p><p> </p><p>"Amazon S3 permissions: When you create your role, you specify Amazon S3 permissions. Make sure that your IAM role has access to the S3 bucket that you plan to use in this example. If you do not specify an S3 bucket in this notebook, the default bucket in the AWS Region in which you are running this notebook instance is used. If you do not require granular permissions, you can attach <code><a href="https://console.aws.amazon.com/iam/home#policies/arn:aws:iam::aws:policy/AmazonS3FullAccess" target="_blank">AmazonS3FullAccess</a></code> to your role."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>